Source/Urho3D/CSharp/Managed/Math/Matrix2.cs

```diff
@@ -223,7 +223,9 @@ public static unsafe void BulkTranspose(float* dest, float* src, int count)
         public static readonly Matrix2 ZERO = new Matrix2(0, 0, 0, 0);
 
         /// Identity matrix.
-        public static readonly Matrix2 IDENTITY;
+        public static readonly Matrix2 IDENTITY = new Matrix2(
+            1, 0,
+            0, 1);
```

Comment:
This is exactly same as default constructor. Same is true for other matrix types.